test/members_page_test.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+require_relative './test_helper'
+require_relative '../lib/members_page'
+
+describe MembersPage do
+  around { |test| VCR.use_cassette(File.basename(url), &test) }
+  let(:response)  { MembersPage.new(response: Scraped::Request.new(url: url).response) }
+  
+  describe 'first page' do
+    let(:url) { 'http://www.congreso.es/portal/page/portal/Congreso/Congreso/Diputados/DiputadosTodasLegislaturas' }
+    it 'should contain the expected number of member urls' do
+      response.member_urls.count.must_equal 25
+    end
+
+    it 'should contain the expected url' do
+      response.member_urls.first.must_equal 'http://www.congreso.es/portal/page/portal/Congreso/Congreso/Diputados/BusqForm?_piref73_1333155_73_1333154_1333154.next_page=/wc/fichaDiputado&idDiputado=87&idLegislatura=10'
```

Comment:
The problem with stripping the '_piref' part of the URL out is that when we request the next page without the the '_piref' in the url, we get a page with the members list twice:
![screen shot 2017-03-20 at 15 32 23](https://cloud.githubusercontent.com/assets/4107953/24107964/22e55cf8-0d84-11e7-91dd-8ed335d1b760.png)

I think the decorator should be used on the `PersonMembership` pages though. My plan is to use these URLs to create the `MembershipPage`s.